### PR TITLE
Implementation of Inception V4 (#18)

### PR DESCRIPTION
**Pull request by AlexeyKurakin**

### CODE REVIEW

**Comment by pmenglund** [comment]
Any plans to release the trained model like you did for v3?

http://download.tensorflow.org/models/image/imagenet/inception-2015-12-05.tgz

**Comment by AlexeyKurakin** [comment]
We have release trained checkpoint as well: http://download.tensorflow.org/models/inception_v4_2016_09_09.tar.gz

Link and accuracy on validation set are also provided in README file: https://github.com/tensorflow/models/blob/master/slim/README.md
